provisioning/roles/kuma/files/apache2/all-servers.conf

```diff
@@ -3,15 +3,6 @@
 # Our servers symlink to this file. Do not move it without first coordinating
 # with WebOps.
 
-<Location /static/fonts>
-    # Only enable CORS for fonts when the request is coming from an approved domain
-    Header unset Access-Control-Allow-Origin
-    SetEnvIf Origin "https?://(.*\.(mozilla|allizom)\.(com|org|net))" CORS=$0
-    SetEnvIf Origin "https?://(mdn\.mozillademos\.org)" CORS=$0
-    SetEnvIf Origin "https?://(kuma\.readthedocs\.org)" CORS=$0
-    Header set Access-Control-Allow-Origin %{CORS}e env=CORS
```

Comment:
Hm... I guess I'm not sure about whether we want to remove this. I can restore this locally and verify that it still works once I'm able to test.